src/core/mount-setup.c

```diff
@@ -419,55 +420,27 @@ static int relabel_extra(void) {
          * possible.
          */
 
-        d = opendir("/run/systemd/relabel-extra.d/");
-        if (!d) {
-                if (errno == ENOENT)
-                        return 0;
-
-                return log_warning_errno(errno, "Failed to open /run/systemd/relabel-extra.d/, ignoring: %m");
-        }
+        r = conf_files_list_strv(&files, ".relabel", NULL, CONF_FILES_FILTER_MASKED,
+                                 (const char * const*) STRV_MAKE("/run/systemd/relabel-extra.d/"));
```

Comment:
why not `conf_files_list(…, "/run/systemd/relabel-extra.d/\0")`?

i.e. use the nulstr instead of the strv version here. it's not just easier to read but also quicker

Reply:
also, maybe add a comment that we are not using CONF_DIRS() here on purpose (because we cannot use unlink())

Reply:
I think the comment above is enough...